ui/src/containers/SubOrgForm/index.js

```diff
@@ -76,7 +77,12 @@ class SubOrgForm extends Component {
     const { model } = this.props;
     const modelId = model.get('_id');
     const oldSettings = model.get('settings');
-    const newSettings = oldSettings.set(attr, value);
+    
+    if (!isArray(attr)) {
```

Comment:
Better solution would be to find everywhere this function is called and just ensure it's called with an array, then rename `attr` to `path` here.

Reply:
https://github.com/LearningLocker/learninglocker/pull/1215/commits/cb1c60f3bbb19386d0aec469a1d346885a8a3141 renamed